applicationconfig/client/src/main/java/com/azure/applicationconfig/AzConfigClient.java

```diff
@@ -63,39 +64,38 @@
      * @return an instance of AzConfigClient
      */
     public static AzConfigClient create(String connectionString) {
-        return create(connectionString, new PipelineOptions());
+        return create(
+                connectionString,
+                HttpLogDetailLevel.BASIC);
```

Comment:
My personal stance on newlines like this is that they are fine if required - but in this case I prefer the old code because it all fits on one line and reads fine. Thoughts?

Reply:
I can fit it on one line. iirc, it was longer than 120 characters when I was coding so I had wrapped it before.